test/Microsoft.ML.AutoML.Tests/AutoFitTests.cs

```diff
@@ -113,11 +139,13 @@ public void AutoFitRegressionTest()
             var validationData = context.Data.TakeRows(trainData, 20);
             trainData = context.Data.SkipRows(trainData, 20);
             var result = context.Auto()
-                .CreateRegressionExperiment(0)
+                .CreateRegressionExperiment(experimentTime)
                 .Execute(trainData, validationData,
                     new ColumnInformation() { LabelColumnName = DatasetUtil.MlNetGeneratedRegressionLabel });
 
             Assert.True(result.RunDetails.Max(i => i.ValidationMetrics.RSquared > 0.9));
+
+            Thread.CurrentThread.CurrentCulture = originalCulture;
```

Comment:
Should this be in a finally block, incase the above throws? Or is the thread complete upon failures?

Reply:
Thanks.
#Resolved